sentry_sdk/tracing.py

```diff
@@ -14,7 +14,7 @@
     from typing import Dict
     from typing import Iterator
     from typing import List
-    from typing import Mapping
+    from collections.abc import Mapping
```

Comment:
Small nitpick (feel free to leave unchanged): Now that you changed this line to import from `collections.abc` instead of from `typing`, it is a bit weird that it is surrounded by `typing` imports. Could you reorder this import to be above all the `typing` imports?

On a related note, it would be nice if we could use some kind of automatic Python formatting tool to automatically enforce alphabetical ordering for imports. Something like this exists in Rust, and I find that it helps keep code more organized.

Reply:
Fixed: https://github.com/getsentry/sentry-python/pull/2645/commits/3110117e5e3c63d301ca68719dfa4ae9e3926541